examples/tone_analyzer.v3.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
-const ToneAnalyzerV3 = require('watson-developer-cloud/tone-analyzer/v3');
+//const ToneAnalyzerV3 = require('watson-developer-cloud/tone-analyzer/v3');
+const ToneAnalyzerV3 = require('../tone-analyzer/v3');
```

Comment:
Don't do this. It makes everyone else have to edit the example before they can use it.

Instead, you want to go into the root directory and run `npm link` and then go into the examples directory and run `npm link watson-developer-cloud`. It does basically the same thing as this, but without the code change.

See https://docs.npmjs.com/cli/link